screens/EpisodeInputFlow/EpisodeDisplayScreen.tsx

```diff
@@ -26,21 +26,25 @@ function EpisodeDisplay({ episode, index, deleteCallback }: EpisodeProps) {
   const { width } = useWindowDimensions();
   const offset = useRef(new Animated.Value(0)).current; // Initial value for opacity: 0
 
+  // animate the sliding delete button
   useEffect(() => {
     const toValue = clicked ? -width + 60 : 0;
     Animated.timing(offset, {
       toValue,
       duration: 200,
-      useNativeDriver: false,
+      useNativeDriver: true,
```

Comment:
Wait wasn't this not working before? Does it work now?

Reply:
ya, flipping this was what made it look s m o o t h

Reply:
jk, it seems to be fine now? I'm not sure tbh

Reply:
nah yeah this is the thing that fixed it